stix_shifter_modules/alertflex/configuration/alertflex_lang.json

```diff
@@ -0,0 +1,49 @@
+{
+    "connection": {
+        "host": {
+            "label": "Management IP address or Hostname",
+            "placeholder": "192.168.1.10",
+            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
```

Comment:
You don't need to mention the bit about IBM Cloud Pak since the host would be needed to make a connection to the data source regardless of what service is using the stix-shifter library.